dll/win32/shell32/utils.cpp

```diff
@@ -9,6 +9,56 @@
 
 WINE_DEFAULT_DEBUG_CHANNEL(shell);
 
+static BOOL OpenEffectiveToken(DWORD DesiredAccess, HANDLE *phToken)
+{
+    BOOL ret;
+
+    if (IsBadWritePtr(phToken, sizeof(*phToken)))
```

Comment:
This is a baroque function, not sure it has to be used there. OpenThreadToken/OpenProcessToken should already do validation checks and return a suitable error there.

Reply:
Fixed.